services/gundeck/test/unit/MockGundeck.hs

```diff
@@ -2,6 +2,7 @@
 {-# LANGUAGE RecordWildCards #-}
```

Comment:
I'm wondering why you haven't had to touch any test to validate the changed behavior. Though, it can well be that I just don't understand the impact of the changes to the mock well enough. :thinking:

Reply:
The test is just comparing the `MockState` after running the mock type instance and the real type instance with the `MockGundeck`. So, changing the mock instance to do what we expect was enough to make the test correct.